### PR TITLE
Codestyle

### PR DESCRIPTION
**Pull request by agronholm**
First batch of code style fixes.

### CODE REVIEW

File: pytypes/stubfile_2_converter.py [hunk 1]
```diff
@@ -133,12 +147,12 @@ def convert(in_file, out_file = None):
 		os.makedirs(directory)
 
 	with open(out_file, 'w') as out_file_handle:
-		lines = ["'''",
+		lines = [""""",
```
Comment (Stewori) [inline pytypes/stubfile_2_converter.py]: This line looks broken. Should rather be '"""'.

File: pytypes/stubfile_2_converter.py [hunk 2]
```diff
@@ -133,12 +147,12 @@ def convert(in_file, out_file = None):
...
 				'Python 2.7-compliant stubfile of ',
 				in_file,
 				'with MD5-Checksum: '+checksum,
 				'This file was generated by pytypes. Do not edit directly.',
-				"'''",
+				""""",
```
Comment (Stewori) [inline pytypes/stubfile_2_converter.py]: This one too (Should rather be '"""'.)

**Comment by agronholm** [comment]
Fixed.

**Comment by agronholm** [comment]
One big issue seems to be that the test suite does not work on Python 2. Can you fix this before I proceed further?

File: pytypes/stubfile_2_converter.py [hunk 3]
```diff
@@ -147,12 +147,12 @@ def convert(in_file, out_file = None):
 		os.makedirs(directory)
 
 	with open(out_file, 'w') as out_file_handle:
-		lines = [""""",
+		lines = ["'''",
```
Comment (Stewori) [inline pytypes/stubfile_2_converter.py]: I didn't mean you to revert it. Actually the generated file should have PEP 8 compliant docstring.
Just `"""""` wasn't suitable. Should have been `'"""'`
(Same below)

**Comment by Stewori** [comment]
> One big issue seems to be that the test suite does not work on Python 2. Can you fix this before I proceed further?

That's strange. For me it works with Python 2. Please open a separate issue for that, including the failure output and your platform. Fixing that has indeed high priority. Note that for Python 2 the typing module isn't part of stdlib and must be installed explicitly (e.g. via pip); just in case you overlooked that.

**Comment by agronholm** [comment]
Having followed the development of the typing library, I was under the impression that Python 2 support had already been dropped. Hence the confusion.

**Comment by Stewori** [comment]
Looks like the python2 folder still exists there:
https://github.com/python/typing/tree/master/python2

**Comment by agronholm** [comment]
Looks like py2 development got resumed. Okay then. Could you come to IRC? I prefer that to back and forth over github.

**Comment by Stewori** [comment]
Sorry, was AFK. Now online at IRC #jython. Anyway. If test_typechecker really fails for you on Python 2, we should have an issue here to keep track.

**Pull request merged**